templates/egressfirewall.go

```diff
@@ -29,6 +29,12 @@ var EgressFirewallTemplate = ovnv1.EgressFirewall{
 				},
 				Type: ovnv1.EgressFirewallRuleAllow,
 			},
+			{
+				To: ovnv1.EgressFirewallDestination{
+					DNSName: "ingest.us-south.monitoring.cloud.ibm.com",
```

Comment:
- pls factor out this value in a const and use at other places
- if there's any mapping b/n AWS regions & IBM regions we could potentially set this value dynamically
- wherever the const will be defined, pls add a comment that it needs to be constructed dynamically at a later point

Reply:
Added a constant,
will look into dynamic construction as a future TODO and added a comment for the same